modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/OpenAPIResourceTest.java

```diff
@@ -311,25 +322,29 @@ public void testGetOpenAPIWithSystemObjectRelationship() throws Exception {
 			"relationMToM", false,
 			ObjectRelationshipConstants.TYPE_MANY_TO_MANY, null);
 
+		String jsonObject = HTTPTestUtil.invokeToJSONObject(
```

Comment:
Do you need to change this?

Reply:
@ccorreagg  Not really, but for debugging in the future is better to have the variable or not?

Reply:
Yes, but if it is only used once, then it is not necessary, as you might debug using the attribute of the method that is being called with the jsonObject.

Try this:

`git log --grep Inline`